content/kubernetes/re-clusters/delete_custom_resources.md

```diff
@@ -11,66 +11,73 @@ aliases: [
 ]
 ---
 
-## Delete a database
+## Delete a database (REDB)
 
 To delete a database managed by the Redis Enterprise Kubernetes operator, run `kubectl delete redb <your-db-name>` from your K8s cluster.
 
 When you delete a database, your data and the REDB custom resource are also deleted.
 
-### Troubleshoot REDB deletion
+## Delete a Redis Enterprise cluster (REC)
 
-The operator attaches a finalizer to the Redis Enterprise database (REDB) object. This makes sure the database is deleted before the REDB custom resource is removed from the K8s cluster.
+To delete a Redis Enterprise cluster managed by the operator:
 
-If the finalizer isn't removed automatically by the operator, you won't be able to delete your REDB resource.
+1. Delete all the databases in your cluster.
 
-If this happens, you can remove the finalizer manually.
+1. Run `kubectl delete rec <your-rec-name>` from your K8s cluster.
 
-{{<warning>}} If you remove the finalizer manually, there is not guarantee that the underlying REC has been deleted. This may cause resource issues and require manual intervention. {{</warning>}}
+When you delete your cluster, your databases and the REC custom resource are also deleted. However, persistent volume claims (PVCs) for your cluster are not deleted in the process. If you want to delete your PVCs, you'll have to delete them manually.
 
-```sh
-kubectl patch redb <your-db-name> --type=json -p \
-    '[{"op":"remove","path":"/metadata/finalizers","value":"finalizer.redisenterprisedatabases.app.redislabs.com"}]'
-```
+## Delete the operator
 
-## Delete a Redis Enterprise cluster (REC)
+To delete the operator from your K8s cluster and namespace, you can delete the operator bundle with:
 
-To delete a Redis Enterprise cluster managed by the operator:
+- `kubectl delete -f bundle.yaml` for vanilla K8s deployments
```

Comment:
the term vanilla here is confusing, could be understood as generic K8s. I would just say non openshift.